Documentation/sources/content-card-ui/api-usage.md

```diff
@@ -0,0 +1,53 @@
+# APIs Usage
+
+This document provides information on how to use the Messaging APIs to receive and display content card views in your application.
+
+## Importing Messaging
+
+To use the Messaging APIs, you need to import the Messaging extension in your Kotlin file.
+
+```kotlin
+import com.adobe.marketing.mobile.Messaging
+```
+
+## APIs
+
+### getContentCardUI
+
+The `getContentCardUI` method retrieves a flow of [AepUI](./public-classes/aepui.md) objects for the provided surface. These AepUI objects provide the user interface for templated content cards in your application.
```

Comment:
nit: Since `AepUI` itself does not provide the user interface, can we reword a bit : `These AepUI objects represent templated content cards whose UI can be rendered using provided card composables`